site/layouts/shortcodes/apiendpoint.html

```diff
@@ -0,0 +1 @@
+{{ .Site.Params.api.localCordinator }}{{ .Site.Params.api.apiEndpoint }}
```

Comment:
nit: newline

Reply:
@benraskin92 Not in this case, as that would be rendered I think, and thus not work inline.